llvm/tools/llvm-objdump/XCOFFDump.cpp

```diff
@@ -67,6 +75,179 @@ void XCOFFDumper::printStrHex(StringRef Name, StringRef Str, uint64_t Value) {
          << ")\n";
 }
 
+void XCOFFDumper::printBinary(StringRef Name, ArrayRef<uint8_t> B) {
+  unsigned OrgWidth = getWidth();
+  setWidth(0);
+  outs() << formatName(Name) << " (" << format_bytes(B) << ")\n";
+  setWidth(OrgWidth);
```

Comment:
There's a lot of `setWidth` etc calls going on here, but I don't see any testing of the formatting for this case? I.e. if you removed one or other of these, I'm not sure you'd get a test failure?

Reply:
> There's a lot of `setWidth` etc calls going on here, but I don't see any testing of the formatting for this case? I.e. if you removed one or other of these, I'm not sure you'd get a test failure?

I think following test it.

`# RUN: llvm-objdump --private-headers %t1_extra 2>&1 | FileCheck %s --check-prefix=EXTRA --match-full-lines --strict-whitespace`

Reply:
> if you strong suggest to remove it, I can remove it.

I was actually suggesting that you should have more testing, with my suggestion being that you could tell if you had sufficient testing by removing a line of code and seeing if a test failed. However, I think your explanation makes sense, thanks, so no need to change things.